src/aposteriori/data_prep/create_frame_data_set.py

```diff
@@ -754,24 +754,39 @@ def process_paths(
 
 def _select_pdb_chain(pdb_path: pathlib.Path, chain: str):
     """
-    Select a chain from a pdb file
+    Select a chain from a pdb file. The chain will remove the original pdb file.
+    At the moment we only support the selection of one chain at the time, meaning
+    if you wanted to selected chain A and B of a PDB, you should write it twice eg.
+    "6FMLA, 6FMLB".
```

Comment:
But aren't you overwriting the file in place, so chain B would be removed when you save chain A. Would it be worth specifying an output directory and then writing the files as "6fml_a.pdb" so you don't delete the original data?

Reply:
So, actually it deletes the original 6fml pdb file. But it saves the new file as "6fmlA.pdb". Have a look at line 784 where we build the output file name.

Reply:
Sorry I missed that.